Makefile

```diff
@@ -338,24 +396,3 @@ endif
 ifdef main
 	make publish_langflow
 endif
-
-patch: ## bump the version in langflow and langflow-base
```

Comment:
I think we should keep this.

Reply:
Of course, my mistake, I restored the patch to the Makefile.